pytest_fixtures/component/rh_cloud.py

```diff
@@ -35,13 +35,15 @@ def organization_ak_setup(rhcloud_sat_host, rhcloud_manifest_org):
         purpose_role='test-role',
         auto_attach=True,
     ).create()
-    subscription = rhcloud_sat_host.api.Subscription(organization=rhcloud_manifest_org)
-    # Disabling due to an issue with manifest refreshes. Is this refresh actually needed?
-    # subscription.refresh_manifest(data={'organization_id': rhcloud_manifest_org.id})
-    default_subscription = subscription.search(
-        query={'search': f'name="{DEFAULT_SUBSCRIPTION_NAME}"'}
-    )[0]
-    ak.add_subscriptions(data={'quantity': 2, 'subscription_id': default_subscription.id})
+    if (
+        rhcloud_sat_host.api.Organization(id=rhcloud_manifest_org.id).read().simple_content_access
+        is False
+    ):
+        subscription = rhcloud_sat_host.api.Subscription(organization=rhcloud_manifest_org)
```

Comment:
Is this failing anywhere else? I see `rhcloud_manifest_org` uses `module_entitlement_manifest`, so this block will always run. Should we have that use `module_sca_manifest` instead and then we don't need any of this code?

Reply:
yeah, let's go with `module_sca_manifest`, let's get rid of that code :D

Reply:
Done